packages/tools/package/test/module/changelog/changelog.utils.test.js

```diff
@@ -0,0 +1,66 @@
+const changelogUtils = require('@webex/package-tools/dist/module/commands/changelog/changelog.utils');
+const changelogExecutor = require('@webex/package-tools/dist/module/commands/changelog/changelog.executor');
```

Comment:
Why do we have to require this way with a complete path? Does relative path not work? Is there some error?

Reply:
Yes. Im unable to import them relatively. I have used this also to maintain coding convention in packag-tools. This is exactly how utils have been tested in this package.